agent/lib/kontena/workers/container_health_check_worker.rb

```diff
@@ -43,7 +43,7 @@ def check_status
       else
         msg = check_tcp_status(ip, port, timeout)
       end
-      rpc_client.async.notification('/containers/health', [msg])
+      rpc_client.async.request('/containers/health', [msg])
```

Comment:
I hope we have some reasonable bound on the number of healthchecks... but it's going to be less than the logs, for sure.